contracts/factory/EVMScriptRegistryFactory.sol

```diff
@@ -25,12 +25,13 @@ contract EVMScriptRegistryFactory is AppProxyFactory, EVMScriptRegistryConstants
 
         ACL acl = ACL(_dao.acl());
 
-        acl.createPermission(this, reg, reg.REGISTRY_MANAGER_ROLE(), this);
+        acl.createPermission(this, reg, reg.REGISTRY_ADD_EXECUTOR_ROLE(), this);
 
         reg.addScriptExecutor(baseCallScript);     // spec 1 = CallsScript
 
-        acl.revokePermission(this, reg, reg.REGISTRY_MANAGER_ROLE());
-        acl.setPermissionManager(_root, reg, reg.REGISTRY_MANAGER_ROLE());
+        // Clean up the permissions
+        acl.revokePermission(this, reg, reg.REGISTRY_ADD_EXECUTOR_ROLE());
+        acl.removePermissionManager(reg, reg.REGISTRY_ADD_EXECUTOR_ROLE());
```

Comment:
@izqui Note that I've changed the factory to remove the permission's manager to reset it entirely, rather than leaving it dangling with root as the permission manager.

Reply:
Thanks for pointing it out, I think it is a good idea. Created an issue as it requires changes in to kits: https://github.com/aragon/dao-kits/issues/23